ServiceApp/server/src/utils/credentialHelper.ts

```diff
@@ -29,7 +31,78 @@ export interface IUser {
     address: string;
 }
 
-export function createNewUser(name: string = '', role: string = '', location: string = ''): Promise <IUser> {
+export function createNewUserC2(name: string = '', role: string = '', location: string = ''): Promise<IUser> {
+    return new Promise<IUser>(async (resolve, reject) => {
+        const clientConfig = {
+            network: "main",
+            node: "https://chrysalis-nodes.iota.org:443",
+        }
+        try {
+
+            // Create a new DID Document with the KeyPair as the default authentication method
+            //@ts-ignore
+            const { doc, key } = new Document(KeyType.Ed25519, clientConfig.network);
```

Comment:
On line 80 you are using `Network.mainnet()` but here you are using a custom network URL. Please be consistent and use the same network everywhere

Reply:
fixed in3c6978c6f75aa8ed7718b8a8b6f9e08a561585a2